docs/README.md

```diff
@@ -59,13 +59,17 @@
 
 - [New Maintainer Checklist](New-Maintainer-Checklist.md)
 - [Maintainers: Avoiding Burnout](Maintainers-Avoiding-Burnout.md)
+
 - [Maintainer Guidelines](Maintainer-Guidelines.md)
-- [Homebrew/linuxbrew-core Maintainer Guide](Homebrew-linuxbrew-core-Maintainer-Guide.md)
+- [`brew` Maintainer Guide](Brew-Maintainer-Guide.md)
+- [Core Maintainer Guide](Homebrew-homebrew-core-Merge-Checklist.md)
+- [Linux Maintainer Guide](Linux-Maintainer-Guide.md)
+- [Cask Maintainer Guide](Cask-Maintainer-Guide.md)
```

Comment:
```suggestion
- [`brew` Maintainer Guide](Homebrew-brew-Maintainer-Guide.md)
- [Core Maintainer Guide](Homebrew-homebrew-core-Merge-Checklist.md)
- [Linux Maintainer Guide](Homebrew-linuxbrew-core-Maintainer-Guide.md)
- [Cask Maintainer Guide](Homebrew-homebrew-cask-Maintainer-Guide.md)
```
Unless, of course, these are the intended filenames for a future commit.

Reply:
Thanks, forgot to update these